CONTRIBUTING.md

```diff
@@ -17,7 +17,14 @@ This guide shows how to build the whole project only.
 ### System Dependencies
 You need to install the following dependencies:
   * Linux or macOS. Windows support is coming soon. <!-- TODO: Link Java roadmap when it is published -->
-  * [JDK 1.8+](https://jdk.java.net/).
+  * [JDK 1.8](https://jdk.java.net/) and JDK 11.0+ toolchains.
```

Comment:
How about to inject a link to the toolchains? 
Also, configuration below starts with something related to `jenv` which is confusing ones who don't use it. (for example, I use sdkman for managind java versions, so what should I do then?)
So I propose to provide clear step-by-step instructions about how to install and configure toolchains or add a link to it.

Reply:
> starts with something related to jenv 

Not necessarily: it specifies to which directory layout these instructions are appropriate.

> (for example, I use sdkman for managind java versions, so what should I do then?)

It does say: "If you don't use jenv, replace with appropriate location" (assuming you have the expected layout).

I agree though that a line with instructions for those who use a different layout could be added — but it is likely we won't use toolchains during development (maybe, we will keep them on CI only to _run tests_ on 8 for client libraries).